File: app/src/main/java/com/hikko/scheduleapp/Activity.java
package com.hikko.scheduleapp;

// Пара
public class Activity {
    public String start;
    public String end;
    public String name;
    public String type;
    public int id;

    public Activity(String name, String type, String start, String end) {
        this.name = name;
        this.type = type;
        this.start = start;
        this.end = end;
        this.id = -1;
    }
    public Activity(String name, String type, String start, String end, int id) {
        this.name = name;
        this.type = type;
        this.start = start;
        this.end = end;
        this.id = id;
    }
}

File: app/src/main/java/com/hikko/scheduleapp/Activity.kt
package com.hikko.scheduleapp

// Пара
class Activity : Cloneable {

    var startTime: String = ""
    var endTime: String = ""
    var name: String = ""
    var type: String = ""
    var cabinet: String = ""
    var id: Int = -1

    public override fun clone(): Activity {
        return super.clone() as Activity
    }

    override fun toString(): String {
        return "{id: $id, type: \"$type\", name: \"$name\", " +
                "cabinet: \"$cabinet\", startTime: \"$startTime\", endTime: \"$endTime\"}"
    }
}